<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication2Class</class>
 <widget class="QMainWindow" name="QtGuiApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="method1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MedianBlur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="method3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="method2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GaussianBlur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish_Bot">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_Bot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上傳圖片</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="img_src">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>531</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="img_after">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>531</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kernel_Input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kernel 數值更改   預設為5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kernel_Check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>確定更改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注意事項: 要先上傳圖片再選方法，kernel值必須是正整數且只能是奇數。</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
